metabase/metabase
# Fix styling for heading cards in dashboard subscriptions

Closes https://github.com/metabase/metabase/issues/31620

### Description

Escape markdown characters in headings cards, and add heading markdown.

### How to verify

Go to any dashboard
Click on edit
Add a heading card and add some text to the heading
Create dashboard subscription
Send dashboard subscription to an email you can view
See heading in plain text


### Demo

<img width="575" alt="image" src="https://github.com/metabase/metabase/assets/34140255/b9cb17a6-3979-401b-b1e3-c736617b0fd9">
